### PR TITLE
add some epic-store games

### PR DESCRIPTION
**Pull request by Dei657**

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -47,6 +47,12 @@
   notes:
     - Displays keyboard icons
 
+- name: Brothers - A Tale of Two Sons
+  platform: epic-store
+  id: Tamarind
+  banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/f730dba64b56c5b89a94bcb8cc3b0fba.jpg
```
Comment (alkazar) [inline gamedb.yaml:53]: Would be better to use the steam banner. An upcoming feature will automatically grab all the other types of artwork from Steam in addition to the horizontal banner.

File: gamedb.yaml [hunk 2]
```diff
@@ -206,6 +234,12 @@
   banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/e3408432c1a48a52fb6c74d926b38886.png
   status: playable
 
+- name: Hello Neighbor
+  platform: epic-store
+  id: Ursus
+  banner: steam: 521890
```
Comment (alkazar) [inline gamedb.yaml:240]: There is an extra space after the 'steam:'

**Comment by Dei657** [comment]
I understood, I will correct it now.

**Comment by alkazar** [comment]
Awesome, thank you so much!

**Pull request merged**